frontend/src/AlertConfiguration/AlertConfigurationComponents/ConfigCard.js

```diff
@@ -32,10 +30,10 @@ export default function ConfigCard(card) {
           variant="h5"
           component="h2"
         >
-          Alert Configuration for "{ card.metric }"
+          Alert Configuration for "{ card.dimension }"
```

Comment:
do you want both metric and dimension in the title?
or do you think just dimension is enough?